broadinstitute/single_cell_portal_core
# Upgrading transitive dependency braces re: CVE-2024-4068 (SCP-5660)

#### BACKGROUND & CHANGES
Identified as a vulnerability by AppSec via Defectdojo: https://defectdojo.dsp-appsec.broadinstitute.org/finding/225003

This gets the deployed version of `braces` to `3.0.3` via upgrades on `sass` and `sentry/vite-plugin`.  There still remains a dev dependency via `jest-enzyme` for an older version, but as this is not deployed it isn't a security risk.

#### MANUAL TESTING
1. Pull branch and run `yarn install`
2. Start server & vite and confirm pages load as normal
